### DISCUSSION TITLE
希望增加每个视频单独记忆字幕状态的功能

### DISCUSSION THREAD

**Discussion by mobeicanyue** [the1812/Bilibili-Evolved#4551] [功能建议 / Ideas]
希望增加每个视频单独记忆字幕状态的功能，遇到那种几十p的视频，每次都得手动开启AI字幕，很难受

**Comment by Geeyun-JY3** [comment]
可能相关：https://github.com/the1812/Bilibili-Evolved/discussions/3745#discussioncomment-3897044
